compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1787,6 +1789,8 @@ private OperatorPrecedence getOpPrecedence(SyntaxKind binaryOpKind) {
                 return OperatorPrecedence.LOGICAL_AND;
             case LOGICAL_OR_TOKEN:
                 return OperatorPrecedence.LOGICAL_OR;
+            case TYPEOF_KEYWORD:
+                return OperatorPrecedence.UNARY;
```

Comment:
According to OperatorPrecedence.java `UNARY(1), //  (+x), (-x), (~x), (!x), (<T>x), (typeof x), (check x), (checkpanic x)`
Therefore, `check` and `checkpanic` cases should also have declared with `OperatorPrecedence.UNARY;` 
but in ballerina-spec document they are not listed in the precedence table.